src/config/examples/main.rs

```diff
@@ -0,0 +1,21 @@
+#[macro_use]
+extern crate amethyst_config;
+extern crate yaml_rust;
+
+use amethyst_config::Element;
+use std::path::Path;
+
+fn main() {
+    let config = amethyst_config::Config::from_file(Path::new("../../config/config.yml"));

```

Comment:
If you use `AsRef<Path>`, like `File::open` [does](http://doc.rust-lang.org/std/fs/struct.File.html#method.open), you could avoid `Path::new()` here.